alacritty/src/config/bindings.rs

```diff
@@ -344,53 +344,57 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
         let modifiers_code = index + 2;
         bindings.extend(bindings!(
             KeyBinding;
-            Key::Up,    [shift: mods.shift, alt: mods.alt, ctrl: mods.ctrl];
+            Key::Insert, [shift: mods.shift, alt: mods.alt, ctrl: mods.ctrl];
```

Comment:
On non-macos platforms we do already have Shift+Insert bound to `PasteSelection`. So I'm not sure what we want to do in this case?

I personally do not care about this `Shift+Insert` binding, but in the current version of the PR this would mean that both bindings are active as far as I can tell, so that could obviously cause issues.